q2_fmt/_peds.py

```diff
@@ -316,38 +284,38 @@ def feature_peds(table: pd.DataFrame, metadata: qiime2.Metadata,
 
     with warnings.catch_warnings():
         warnings.simplefilter("ignore")
-        peds = median_numerator_series/median_denominator_series
+        prdf = median_numerator_series/median_denominator_series
 
-    peds_df['measure'] = peds.values
-    peds_df['recipients with feature'] = median_numerator_series.values
-    peds_df['all possible recipients with feature'] = \
+    prdf_df['measure'] = prdf.values
+    prdf_df['recipients with feature'] = median_numerator_series.values
+    prdf_df['all possible recipients with feature'] = \
         median_denominator_series.values
-    peds_df = peds_df.reset_index()
+    prdf_df = prdf_df.reset_index()
 
-    peds_df['id'].attrs.update({
+    prdf_df['id'].attrs.update({
         'title': "Feature ID",
         'description': ''
     })
-    peds_df['measure'].attrs.update({
+    prdf_df['measure'].attrs.update({
         'title': "Feature PEDS",
```

Comment:
```suggestion
        'title': "Feature PRDF",
```
I'm assuming this title should get updated as well?